### DISCUSSION TITLE
Which mod crashed

### DISCUSSION THREAD

**Discussion by zc1108aaa** [FabricMC/fabric-api#3805] [FAQ]
[crash-2024-05-27_14.20.30-client.txt](https://github.com/FabricMC/fabric/files/15451734/crash-2024-05-27_14.20.30-client.txt)
1
[latest.log](https://github.com/FabricMC/fabric/files/15451736/latest.log)

**Comment by JJplayz565** [comment]
Looks like Nature's Spirit and Woods and Mires
